app/Http/Controllers/Api/V1/Admin/ProjectController.php

```diff
@@ -101,11 +101,12 @@ public function toggleStarter(Request $request, $flag)
     }
 
     /**
-     * Temporary function for clearing db cache data - will be removed in future releases
+     * CUR - 612 => Update existing project rows for is_user_starter flag
```

Comment:
What is this comment?

Reply:
This comment is reference to ticket, it's one time script so I've to remove later. That's why I kept the reference.